src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -193,18 +194,21 @@ class WorkspaceReimburseView extends React.Component {
                 <FullPageNotFoundView shouldShow={_.isEmpty(this.props.policy)}>
                     <Section
                         title={this.props.translate('workspace.reimburse.captureReceipts')}
-                        icon={Illustrations.ReceiptYellow}
+                        icon={Illustrations.MoneyReceipts}
                         menuItems={[
                             {
                                 title: this.props.translate('workspace.reimburse.viewAllReceipts'),
                                 onPress: () => Link.openOldDotLink(`expenses?policyIDList=${this.props.policy.id}&billableReimbursable=reimbursable&submitterEmail=%2B%2B`),
                                 icon: Expensicons.Receipt,
                                 shouldShowRightIcon: true,
                                 iconRight: Expensicons.NewWindow,
+                                iconFill: themeColors.buttonSuccessBG,
```

Comment:
if we're holding for new color theme, you'll want to use `themeColors.success`

Reply:
You can wait on this until the colors are merged, I just wanted to make a note

Reply:
Cool! Will keep this in here for now for testing purposes but can swap it once that's ready to go.